Dockerfile

```diff
@@ -0,0 +1,12 @@
+﻿FROM microsoft/dotnet:3.0-sdk AS build
```

Comment:
lets remove if you've abandoned self contained. I am okay going up to dotnet 3 though